pkg/capture/flow.go

```diff
@@ -53,20 +73,138 @@ func (f *FlowLog) Flows() map[string]*GPFlow {
 	return f.flowMap
 }
 
-// TablePrint pretty prints the flows in a formatted table
+// ParsePacket processes / extracts all information contained in the IP layer received
+// from a capture source and converts it to a hash and flags to be added to the flow map
+func ParsePacket(ipLayer capture.IPLayer, pktType capture.PacketType, pktTotalLen uint32) (epHash capturetypes.EPHash, isIPv4 bool, auxInfo byte, err error) {
```

Comment:
This is mostly unchanged from the previous method acting on a `GPPacket` (except for fixed error variables and compiler hints).